src/routes/MetaDetails/StreamsList/Stream/Stream.js

```diff
@@ -74,41 +116,110 @@ const Stream = ({ className, videoId, videoReleased, addonName, name, descriptio
         }
     }, [props.onClick, profile.settings, markVideoAsWatched]);
 
+    const copyMagneticLinkToClipboard = React.useCallback((event) => {
```

Comment:
It can be just named `copyStreamLink` now that we use `deepLinks.externalPlayer.download`